### ISSUE TITLE
Issue regarding landmark detection

### ISSUE THREAD

**Issue by onzone** [TropComplique/wing-loss#13]
Hey I have trained your network with CelebA data and I have followed the instructions to prepare the dataset for training. But at the inference time I see landmarks are not properly detected(slightly shifted from the original position). Is there any setting in training that I need to change? My training loss had started from around 100 and gone upto 29. Is it right? or should I train more? Can you share a pretrained model?

**Comment by onzone** [comment]
did anyone find anything strange about the loss values?? what is the range of values ??
